admin/webUI/assets/css/custom.css

```diff
@@ -154,6 +154,38 @@ button.btn-lg {
   margin-left: 0;
   margin-right: 0;
 }
+
+/* So that top bar takes up less space.*/
+
+.mdl-layout__header {
+  min-height: 40px;
+}
+
+.mdl-layout__header-row {
+  height: 40px;
+}
+
+.mdl-grid {
+  padding-top: 0px;
+}
+
+.mdl-cell--12-col {
+  margin-top: 0px;
+}
```

Comment:
Oh okay changing this stuff. I think the best way would be to just add custom classes. I was worried I would end up with too many of them.